File: C#/Easy/FizzBuzz.cs
public class Solution
{
    public IList<string> FizzBuzz(int n)
    {
        const int FIZZ = 3;
        const int BUZZ = 5;

        List<string> ans = new List<string>();
        for (int i = 1; i <= n; i++)
        {
            string OneOutput = "";

            if (i % FIZZ == 0)
            {
                OneOutput += "Fizz";
            }
            if (i % BUZZ == 0)
            {
                OneOutput += "Buzz";
            }
            if (OneOutput == "")
            {
                OneOutput += i;
            }
            ans.Add(OneOutput);
        }
        return ans;
    }
}

File: C++/Easy/Fizz Buzz.cpp
#include <string>
#include <vector>
using namespace std;

class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> output;
        string input;
        for (int i = 1; i <= n; i++) {
            if (i % 3 == 0) {
                input += "Fizz";
            }
            if (i % 5 == 0) {
                input += "Buzz";
            }
            if (i % 3 != 0 && i % 5 != 0) {
                input += std::to_string(i);
            }
            output.push_back(input);
            input = "";
        }
        return output;
    }
};